README.md

```diff
@@ -53,6 +55,7 @@ Other Commands
 | | Run the linter only for the backend | `yarn lint` | |
 | | Run the linter for the the backend with results output to xml files | `yarn lint:ci`| |
 | | Run `yarn lint:ci` for both the frontend and backend | `yarn lint:all`| |
+| | Host the open api 3 spec using [redoc](https://github.com/Redocly/redoc) at `localhost:5000` | `yarn docs:serve` | |
```

Comment:
No need to do this now (need to answer questions around authentication first anyway), but in the future I'd like to serve the api docs off of production as well. You don't foresee any issues using this framework to do that, do you?

Reply:
Should work fine, `redoc-cli` can generate a static HTML file that we can use in production (or any other deployed environment)